avocado/utils/configure_network.py

```diff
@@ -200,3 +201,18 @@ def is_interface_link_up(interface):
         log.info("Interface %s link is up", interface)
         return True
     return False
+
+
+def get_ip_address(interface, version='4'):
+    """
+    Get the IP address from a network interface
+    :param interface: name of the interface
+    :param version: IP version, default '4'
+    :return: IP address or False
+    """
+    cmd = "ip -%s -j address show %s" % (version, interface)
+    output = json.loads(process.system_output(cmd).decode("utf-8"))
+    if not output:
+        log.error("Unable to get ip address on interface: %s", interface)
+        return False
+    return output[0]['addr_info'][0]['local']
```

Comment:
@anascko thanks for this, looks like you liked the idea of using json with iproute.
But, this line could raise `KeyError` exception. `try`...`except` is needed here.